src/implementations/twig/components/pagination/pagination.html.twig

```diff
@@ -50,12 +50,14 @@
           {% if _item.aria_label is defined and _item.aria_label is not empty %}
             {% set _link = _link|merge({ aria_label: _item.aria_label }) %}
           {% endif %}
+          {% set _icon = {} %}
           {% if _item.link.icon is defined and _item.link.icon is not empty %}
             {% set _icon = _icon_path is not empty ? _item.link.icon|merge({ path: _icon_path }) : _item.link.icon %}
             {% set _link = _link|merge({ 'icon': _icon }) %}
```

Comment:
This line can be removed, we have two separated variables for the two objects link and icon, this merge does not make sense.